docs/HiddenMarkovModel.rst

```diff
@@ -6,63 +6,80 @@ Hidden Markov Models
-	>>> s1 = State( Distribution( NormalDistribution(5, 1) ) )
-	>>> s2 = State( Distribution( NormalDistribution(1, 7) ) )
-	>>> s3 = State( Distribution( NormalDistribution(8, 2) ) )
-	>>> model = HiddenMarkovModel()
-	>>> model.add_states(s1, s2, s3)
-	>>> model.add_transition(model.start, s1, 1.0)
-	>>> model.add_transition(s1, s1, 0.7)
-	>>> model.add_transition(s1, s2, 0.3)
-	>>> model.add_transition(s2, s2, 0.8)
-	>>> model.add_transition(s2, s3, 0.2)
-	>>> model.add_transition(s3, s3, 0.9)
-	>>> model.add_transition(s3, model.end, 0.1)
-	>>> model.bake()
-
-Models built in this manner must be explicitly "baked" at the end. This finalizes the model topology and creates the internal sparse matrix which makes up the model. This removes "orphan" parts of the model, normalizes all transitions to make sure they sum to 1.0, stores information about tied distributions, edges, and pseudocounts, and merges unneccesary silent states in the model for computational efficiency. This can cause the `bake` step to take a little bit of time. If you want to reduce this overhead and are sure you specified the model correctly you can pass in `merge="None"` to the bake step to avoid model checking.
+	from pomegranate import *
+	s1 = State( Distribution(NormalDistribution(5, 1)))
+	s2 = State( Distribution(NormalDistribution(1, 7)))
+	s3 = State( Distribution(NormalDistribution(8, 2)))
+	model = HiddenMarkovModel()
+	model.add_states(s1, s2, s3)
+	model.add_transition(model.start, s1, 1.0)
+	model.add_transition(s1, s1, 0.7)
+	model.add_transition(s1, s2, 0.3)
+	model.add_transition(s2, s2, 0.8)
+	model.add_transition(s2, s3, 0.2)
+	model.add_transition(s3, s3, 0.9)
+	model.add_transition(s3, model.end, 0.1)
+	model.bake()
+
+Initially it may seem that the first method is far easier due to it being fewer lines of code. However, when building large sparse models defining a full transition matrix can be cumbersome, especially when it is mostly 0s.
+
+Models built in this manner must be explicitly "baked" at the end. This finalizes the model topology and creates the internal sparse matrix which makes up the model. This step also automatically normalizes all transitions to make sure they sum to 1.0, stores information about tied distributions, edges, pseudocounts, and merges unneccesary silent states in the model for computational efficiency. This can cause the `bake` step to take a little bit of time. If you want to reduce this overhead and are sure you specified the model correctly you can pass in `merge="None"` to the bake step to avoid model checking.
+
+The second way to initialize models is to use the ``from_samples`` class method. The call is identical to initializing a mixture model.
+
+.. code-block:: python
+
+	from pomegranate import *
+	model = HiddenMarkovModel.from_samples(NormalDistribution, n_components=5, X=X)
+
+Much like a mixture model, all arguments present in the ``fit`` step can also be passed in to this method.
+
```

Comment:
Sorry for such a cryptic comment.  When I read more carefully the Bayes comment about IndependentComponentsDistribution vs MultivariateGaussianDistribution I see it doesn't apply to the question.

I was motivated by a mention in an email thread that you could initialize a model parametrically with starting multivariatedists/covars/transitions as a starting point and then adjust the model from that starting point using the data.  Is that possible, does it make sense, and, if so, where would it fit into the documentation?